I opened it in google sheets and it got an error

```
react-dom.production.min.js:125 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=ScriptError%3A%20Authorization%20is%20required%20to%20perform%20that%20action.&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Nc (react-dom.production.min.js:72)
    at react-dom.production.min.js:81
    at T (react-dom.production.min.js:95)
    at zj (react-dom.production.min.js:230)
    at Th (react-dom.production.min.js:152)
    at tj (react-dom.production.min.js:152)
    at Te (react-dom.production.min.js:146)
    at react-dom.production.min.js:61
    at unstable_runWithPriority (react.production.min.js:25)
    at Da (react-dom.production.min.js:60)
Me @ react-dom.production.min.js:125
react-dom.production.min.js:72 Uncaught (in promise) Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=ScriptError%3A%20Authorization%20is%20required%20to%20perform%20that%20action.&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Nc (react-dom.production.min.js:72)
    at react-dom.production.min.js:81
    at T (react-dom.production.min.js:95)
    at zj (react-dom.production.min.js:230)
    at Th (react-dom.production.min.js:152)
    at tj (react-dom.production.min.js:152)
    at Te (react-dom.production.min.js:146)
    at react-dom.production.min.js:61
    at unstable_runWithPriority (react.production.min.js:25)
    at Da (react-dom.production.min.js:60)
    
    ```

--- a/googlesheets/src/client/sidebar/components/Sidebar.jsx
+++ b/googlesheets/src/client/sidebar/components/Sidebar.jsx
@@ -48,7 +48,7 @@ export default function Sidebar(props) {
 
   // snackbar alert
   const [alertOpen, setAlertOpen] = React.useState(false);
-  const [alertMessage, setAlertMessage] = React.useState(false);
+  const [alertMessage, setAlertMessage] = React.useState('');
   const handleAlertClose = (event, reason) => {
     if (reason === 'clickaway') {
       return;
@@ -67,7 +67,7 @@ export default function Sidebar(props) {
       }
       setGetting(false);
     }).catch(function(err){
-      setAlertMessage(err);
+      setAlertMessage('Unable to get your saved commands. Please try again.');
       setAlertOpen(true);
       setGetting(false);
     });    
@@ -83,6 +83,7 @@ export default function Sidebar(props) {
     setRunning(true);
     serverFunctions.runCommand(name).then(setRunning(false)).catch(function(err){
       setRunning(false);
+      setAlertMessage('Unable to run your command. Please try again.');
     });
   }
 
@@ -134,7 +135,7 @@ export default function Sidebar(props) {
       }
       setSaving(false);
     }).catch(function(err){
-      setAlertMessage(err);
+      setAlertMessage('Unable to save your commands. Please try again.');
       setAlertOpen(true);
       setSaving(false);
     });

